README.md

```diff
@@ -205,6 +211,19 @@ The Diffie-Hellman group should have at least 1024 bit and can be created with t
 ```
 # openssl dhparam -out /etc/exim4/dhparam.pem 2048
 ```
+In the transport section, you need to modify the user and group to the exim-user `system_aliases`-transport (around line 800):
+```
+system_aliases:
+  driver = redirect
+  allow_fail
+  allow_defer
+  data = ${lookup{$local_part}lsearch{/etc/aliases}}
+  user = <exim-user>
+  group = <exim-group>
+  file_transport = address_file
+  pipe_transport = address_pipe
```

Comment:
To improve the readability of the documentation, I'd like to mark debian, centOS or FreeBSD specific things. Not sure if we can do this by color or different boxes. So if you install on debian, you could ignore just all the other instructions.
Problem could be that we don't the precise instruction of each step in every system.